tmt/steps/provision/mrack.py

```diff
@@ -1039,6 +1077,130 @@ class BeakerGuestData(tmt.steps.provision.GuestSshData):
              """,
     )
 
+    bootc_user: Optional[str] = field(
+        default=None,
+        option=('--bootc-user'),
+        metavar='BOOTC_USER',
+        help="""
+             Specify bootc user.
```

Comment:
And what is "bootc user"? `--user` is documented as "A username to use for all guest operations.", so I'd expect `--bootc-user` help text (and other options, e.g. `--botc-password`) tell what they are for. Assume I know nothing about "bootc".